client/state/gutenberg-fse-settings/init.ts

```diff
@@ -0,0 +1,34 @@
+import { useQuery, UseQueryResult } from 'react-query';
+import wpcom from 'calypso/lib/wp';
+
+export const makeQueryKey = ( siteId: string ): string[] => [ 'gutenbergFSESettings', siteId ];
+
+export type SuccessResponse< T > = {
+	body: T;
+	headers: Record< string, string >;
+	status: 200 | 204;
+};
```

Comment:
Do we have an interface like that already? I could not find it.

Reply:
It's very strange that you even need this interface and need to work with the `{ body, headers, status }` structure. The `wpcom.req.get` code should unwrap this "HTTP envelope" and return only the `body` in the promise.